deployment/scripts/postgresql/postgresql_init.sql

```diff
@@ -0,0 +1,29 @@
+/* Entrypoint script for postgres container to set up databases and users for 
+docker-compose setup */
```

Comment:
Is there a PR for the docker-compose file in indexd?

Reply:
This was pulled in from the [compose-services](https://github.com/uc-cdis/compose-services/blob/master/scripts/postgres_init.sql) repo which already includes a [docker-compose.yaml](https://github.com/uc-cdis/compose-services/blob/master/docker-compose.yml#L22).

I can clean up the comment or just add a note in the comments about how it's pulled in from the compose-services repo?

Reply:
Updated the comments here.